components/automate-chef-io/content/docs/data-lifecycle.md

```diff
@@ -217,23 +217,23 @@ curl -s -H "api-token: $TOKEN" -X PUT --data "@config.json" https://{{< example_
 All jobs have the following options:
 
 * `recurrence` (string) - A recurrence rule that determines how often, at what interval, and when to initially start a scheduled job. Any valid recurrence rule [as defined in section 4.3.10 of RFC 2445](https://www.ietf.org/rfc/rfc2445.txt) is valid in this field.
-* `disabled` (bool) - Whether or not this job should be enabled
+* `disabled` (bool) - True or false if this job is an enabled job.
```

Comment:
```suggestion
* `disabled` (bool) - True or false if this job is enabled.
```

Reply:
I'll allow the passive tense in this case. I was trying to avoid using "is."

Reply:
oh i see, i found the original a bit difficult to parse

Reply:
You are right that it's hard to parse. I'm changing it back regardless :)